### PR TITLE
Dird-Client: Implement `config` command and exception for new restriction

### PR DESCRIPTION
**Pull request by nissou91**
* Add an exception to catch when `config` endpoint is being accessed by other tenants rather than the `master_tenant`,
* Add `config` and `DirdCommand` commands to `dird-client`,
* Add `tenant_uuid` to `get` method's header in: `config` and `status` commands.

### CODE REVIEW

**Comment by nissou91** [comment]
recheck

File: wazo_dird_client/__init__.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright 2014-2018 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2014-2022 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from wazo_dird_client.client import DirdClient as Client
-
-__all__ = ['Client']
+from wazo_dird_client.client import DirdClient as Client  # noqa
```
Comment (fblackburn1) [inline wazo_dird_client/__init__.py:5]: Why this change?

File: wazo_dird_client/commands/config.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2022 The Wazo Authors  (see the AUTHORS file)
```
Comment (fblackburn1) [inline wazo_dird_client/commands/config.py:2]:
```suggestion
# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
```

File: wazo_dird_client/exceptions.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
```
Comment (fblackburn1) [inline wazo_dird_client/exceptions.py:4]:
```suggestion
# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
# SPDX-License-Identifier: GPL-3.0-or-later
```

**Review by fblackburn1 — CHANGES_REQUESTED** [review]

File: wazo_dird_client/__init__.py (hunk 1 above)
Comment (nissou91) [reply wazo_dird_client/__init__.py:5]: It was giving me `cannot import exceptions from wazo_dird_client`; and when I change this code, the import was working.

Comment (nissou91) [reply wazo_dird_client/__init__.py]: Done

File: wazo_dird_client/exceptions.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
```
Comment (fblackburn1) [inline wazo_dird_client/exceptions.py:5]:
```suggestion
```suggestion

```
```

Comment (fblackburn1) [inline wazo_dird_client/exceptions.py:5]:
```suggestion
```

Comment (fblackburn1) [inline wazo_dird_client/exceptions.py:5]:
```suggestion
```

File: wazo_dird_client/__init__.py [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright 2014-2018 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
```
Comment (fblackburn1) [inline wazo_dird_client/__init__.py:2]: rebase and don't update copyright if no code change

Comment (fblackburn1) [inline wazo_dird_client/__init__.py:2]: rebase and don't update copyright if no code change

**Review by fblackburn1 — CHANGES_REQUESTED** [review]
Add example in README.md
